lib/common.js

```diff
@@ -5,12 +5,62 @@ const expressBrowserify = require('express-browserify')
 const { makeTextureAtlas } = require('./atlas')
 const { prepareBlocksStates } = require('./models')
 
+function loadData (versionType, majorVersion) {
+  const dataSource = require('../node_modules/minecraft-data/minecraft-data/data/dataPaths')
```

Comment:
What if mcData is not installed in `../node_modules/` eg it is installed globally ?

Reply:
It might be better to expose data sources alongside directly, then. It would fix relative file path issues.

Reply:
Why is this needed ?
Depending on the internal file structure of a package is not good. This is not exposed and is not part of the API, can change at any time.

Reply:
I can hide these functions behind some exposed API and all that if I update minecraft-data. The big problem from from line 58. `workerSrc.browserify.require` takes an absolute file path to expose. So not sure how I'd make that future proof.